src/node_report.h

```diff
@@ -0,0 +1,168 @@
+#ifndef SRC_NODE_REPORT_H_
+#define SRC_NODE_REPORT_H_
+
+// #include "nan.h"
+#include "v8.h"
+#include <uv.h>
+#include <node.h>
+#include <node_buffer.h>
+#include <node_object_wrap.h>
+#include <algorithm>
+#include <cstring>
+#include <climits>
+#include <cstdlib>
+#include <utility>
+#if defined(_MSC_VER)
+# pragma warning(push)
+# pragma warning(disable : 4530)
+# include <queue>
+# include <string>
+# include <vector>
+# pragma warning(pop)
+#else
+# include <queue>
+# include <string>
+# include <vector>
+#endif
+
+
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <iostream>
+#include <sstream>
+#include <iomanip>
+
+#ifdef _WIN32
+#include <time.h>
+#else
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/time.h>
+#endif
+
+namespace nodereport {
+
+using v8::Isolate;
+using v8::Local;
+using v8::Message;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::Object;
+using v8::Number;
+using v8::String;
+using v8::Value;
+using v8::StackTrace;
+using v8::StackFrame;
+using v8::MaybeLocal;
```

Comment:
We don’t use `using` statements in header files

Reply:
done, thanks.